### PR TITLE
Modifed Minions to flip ignition switch pins input

### PR DESCRIPTION
**Pull request by NathanielDelgado**
# Quality Assurance Checklist 
Modified minions read pin function to flip the input of ignition switch so that we don't have to flip manually during application code.

## Requirements 
- [x] Followed Coding Style Guide 
- [x] Presented/discussed in some capacity with others on the Controls team
- [x] Code Build checks pass 
- [x] No merge conflicts 
- [x] Software feature has documentation for it updated in both ReadTheDocs and the comments
- [x] Software feature has documentation for it updated
- Testing
    - [ ] Software feature has test associated with it
    - [ ] Test provides useful information and uses relevant data to accurately represent Controls 
    - [ ] Tested on hardware
    - NOTE: If test file already exists, use that one
- [x] If applicable, have you added the new feature to main.c?
- [x] Tagged appropriate issue ticket on this PR
- [x] Did you have fun?

## Things to Consider 
- [x] Even if the above are checked, is this the best way of writing my code? 
    - It's possible to write code that works, but are there ways to make code more efficient?

### CODE REVIEW

File: Apps/Src/ReadCarCan.c [hunk 1]
```diff
@@ -295,8 +295,8 @@ void TurnMotorControllerPbcOff(void) {
  * @param None
  */
 void UpdatePrechargeContactors(void) {
-    arr_ign_status = (!MinionsRead(kIgn1));
-    mc_ign_status = (!MinionsRead(kIgn2));
+    arr_ign_status = (MinionsRead(kIgn1));
+    mc_ign_status = (MinionsRead(kIgn2));
```
Comment (diyarajon) [inline Apps/Src/ReadCarCan.c:299]: If you want to, the parenthesis around the function could be removed.

File: Apps/Src/DebugDump.c [hunk 2]
```diff
@@ -36,7 +36,7 @@ void TaskDebugDump(void *p_arg) {
             bool pin_state = MinionsRead(pin);
             // Ignition pins are negative logic, special-case them
```
Comment (diyarajon) [inline Apps/Src/DebugDump.c:37]: We don't need this comment here anymore

File: Drivers/Src/Minions.c [hunk 3]
```diff
@@ -22,9 +22,14 @@ void MinionsInit(void) {
 
 bool MinionsRead(Pin pin) {
     if ((kPininfoLut[pin].direction == kInput)) {
+        if (pin == kIgn1 || pin == kIgn2) {
```
Comment (diyarajon) [inline Drivers/Src/Minions.c:25]: Maybe add the comment here:          // Ignition pins are negative logic, special-case them

**Review by diyarajon — APPROVED** [review]
Looks good

**Review by KnockbackNemo — APPROVED** [review]
Looks good!

**Comment by NathanielDelgado** [comment]
destroyed

**Pull request closed without merging**